src/components/HomeSearchBox/index.tsx

```diff
@@ -6,9 +6,29 @@ const HomeSearchBox = () => {
   const { t } = useTranslation('home')
   const router = useRouter()
   const [searchValue, setSearchValue] = useState<string>('')
+  const [active, setActive] = useState(0)
+  const [filtered, setFiltered] = useState<string[]>([])
+  const [isShow, setIsShow] = useState(false)
 
   const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    setSearchValue(event.target.value)
+    const suggestions : string[] = ['Alabama','Alaska','American Samoa', 'Bubble Holmes', 'Contra Hits', 'Amazon'];
+    const searchValue = event.currentTarget.value
+    const newFilteredSuggestions = suggestions.filter(
+      (suggestion: string) =>
+        suggestion.toLowerCase().indexOf(searchValue.toLowerCase()) > -1
```

Comment:
A single line is easier to read, isn't it?
```suggestion
      (suggestion: string) => suggestion.toLowerCase().indexOf(searchValue.toLowerCase()) > -1
```